requirements-fmt.txt

```diff
@@ -2,14 +2,14 @@
 # This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile requirements-fmt.in
+#    pip-compile ./requirements-fmt.in
```

Comment:
I think you want to run `pip-compile` w/o `./` so this change is not registered and to avoid renovate to come and change it everytime. Also, a tiny comment similar to https://github.com/canonical/notebook-operators/pull/362#pullrequestreview-2028508456.